Dup2 is not available on aarch64

need to be upstreamed

diff --git a/cmd/util.go b/cmd/util.go
index 40680f2..fb21ac1 100644
--- a/cmd/util.go
+++ b/cmd/util.go
@@ -446,7 +446,7 @@ func NewPager(fs *flag.FlagSet) *Pager {
 	})
 
 	savedStdout, _ := syscall.Dup(syscall.Stdout)
-	_ = syscall.Dup2(int(pw.Fd()), syscall.Stdout)
+    _ = syscall.Dup3(int(pw.Fd()), syscall.Stdout, 0)
 
 	return &Pager{
 		proc:   proc,
@@ -456,7 +456,7 @@ func NewPager(fs *flag.FlagSet) *Pager {
 
 func (pager *Pager) Close() {
 	if pager.stdout > 0 {
-		_ = syscall.Dup2(pager.stdout, syscall.Stdout)
+		_ = syscall.Dup3(pager.stdout, syscall.Stdout, 0)
 		_ = syscall.Close(pager.stdout)
 	}
 	if pager.proc != nil {
